test/TestProjects/MgmtMockAndSample/samples/Generated/Samples/Sample_DeletedManagedHsmCollection.cs

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Why is the sample generated in the 'samples' folder rather than in the 'tests' folder?

Reply:
these are removed now.